js/components/DashTabs.js

```diff
@@ -155,7 +155,18 @@ class DashTabs extends React.Component {
           loadedManifest: ms.manifest,
           targetSlot: targetSlot,
           viewType: "ImageView",
-          sidePanel: false
+          sidePanel: false,
+          displayLayout: false,
+          bottomPanel: false,
+          canvasControls: {
+            annotations: {
+              annotationLayer: false,
+              annotationCreation: false,
+            },
+            imageManipulation: {
+              manipulationLayer: false,
```

Comment:
Should we leave the manipulation layer on? I could maybe see some cases where someone might want to adjust brightness or contrast to try to see something more effectively

Reply:
I'm happy either way -- I had thought the preference was towards removing as much as possible (à la 'zen mode').  Happy for you to take that call if you have a strong feeling for or against.

Reply:
Sounds good. MP hasn't ever mentioned those tools, but I think it's good to leave them in for now, mention them to him, and then we can remove if he wants them gone.

Reply:
Done -- thanks!